google-account-plugin/src/com/google/gct/idea/AccountPluginInitializationComponent.java

```diff
@@ -63,13 +53,13 @@ void enableFeedbackUtil() {
    * and is therefore not subscribed and listening.
    */
   @VisibleForTesting
-  void enableUsageTracking() {
+  void maybeEnableUsageTracking() {
```

Comment:
I don't like the name prefixed with "maybe". Sounds indeterminate. Perhaps enableUsageTrackingIfPossible